src/ui/component/dayPicker/dayPicker.tsx

```diff
@@ -39,20 +43,78 @@ export const DayPicker: FC<DayPickerProps> = ({
 
   const getLocaleFromLocaleString = useMemo(() => localeMap.get(locale) ?? enUS, [locale])
 
+  const disabledDays = useMemo(() => {
+    const startOfFromYear = new Date(fromYear, 0, 1)
+    const endOfToYear = new Date(toYear, 11, 31)
+
+    switch (type) {
+      case 'rangeFrom':
+        return toDate ? [{ after: toDate, before: endOfToYear }] : []
+      case 'rangeTo':
+        return fromDate ? [{ after: startOfFromYear, before: fromDate }] : []
+      case 'single':
+        return []
```

Comment:
It means that, if I want to use this component as a `single` one, I can't disable any days?

What if I want, as an example, to disable in the calendar the holiday periods?

Reply:
Yes, currently the component doesn't allow to disable arbitrary period, either as 'range' or 'single'.

This is a feature that we can definitely add, by providing a dedicated props.

Do we need this feature in this first version? We can easily iterate on it if needed.

Reply:
The feature has been added in [c952eda](https://github.com/okp4/dataverse-portal/pull/463/commits/c952edae323f25b7790f3132c135c4251b16239d).

cc @lolottetheclash